docs/src/main/sphinx/release/release-397.md

```diff
@@ -0,0 +1,46 @@
+# Release 397 (21 Sep 2022)
+
+## General
+
+* Fix incorrect parsing of invalid values in cast from `varchar` to `timestamp`. ({issue}`14164`)
+* Fix potential incorrect results for queries where all data scanned from tables
+  was processed in the leaf stage and was not passed downstream. ({issue}`14168`)
+* Fix `EXPLAIN (TYPE IO)` query failure for certain queries with redundant table
+  scans. ({issue}`10398`)
+
+## JDBC driver
+
+* Fix potential memory leak when cancelling statements. ({issue}`14176`)
+
+## Delta Lake connector
+
+* Add `parquet.optimized-writer.enabled` as the new configuration property for
```

Comment:
Do we need to add if it defaults to true or false in this context?

Reply:
The defaults are the same as the existing property, only the name has changed